pro/src/screens/VenueForm/VenueFormScreen.tsx

```diff
@@ -35,25 +42,62 @@ const VenueFormScreen = ({
   venue,
   providers,
 }: IVenueEditionProps): JSX.Element => {
-  const onSubmit = () => {
-    alert('todo submit form !')
-  }
+  const history = useHistory()
+  const notify = useNotification()
+  const { currentUser } = useCurrentUser()
+  const [isSiretValued, setIsSiretValued] = useState(true)
```

Comment:
`isSiretValued` est utilisé pour savoir qu'elle champs afficher/valider entre siret et commentaire.

```suggestion
  const [isSiretValued, setIsSiretValued] = useState(!!venue?.siret)
```

Reply:
Ça ne semblait pas fonctionner au moment des tests. J'ai préféré ne pas passer trop de temps sur le bug, et terminer le ticket.